### DISCUSSION TITLE
[Feature] jtxBoard Web Version

### DISCUSSION THREAD

**Discussion by zunami** [TechbeeAT/jtxBoard#1256] [Ideas]
Hello!

I think your app and possibilities are great.
but for large-scale editing, reworking and working efficiently on the computer is a big obstacle.
**I would therefore like a web version from you**. large view, fine-tuning and large data entries.

i am in the commercial field service and am happy to be able to enter my data there. but i always have large entries that i can't finish on my smartphone.

**Comment by patrickunterwegs** [comment]
Hey @zunami , 
thank you for your feedback :-)
I understand the desire for a web-version - or in general a browser-application that offers extensive support for journals, notes and tasks that are interoperable with the iCalendar standard. 
But I developed jtx Board primarily as an Android app. This also means that it uses many functionalities/libraries that are only available within Android. Bringing app functionality to the web would require almost as much work as making something new from scratch. 
Currently I'm working on jtx Board next to a full time job, so I don't have the time and energy to start working on something new. I will reduce my working hours again, but I can't promise that I can work on such a web app any time soon. It would be great if somebody else could make a web project independent of jtx Board though. I hope you understand.

For now I am moving this request to discussions.

**Reply by balporig** [reply]
Kotlin Multiplatform #920 looks like the way to go
